addons/account/views/report_invoice.xml

```diff
@@ -350,9 +350,11 @@
                     </t>
                     <t t-else="">
                         <td>
-                            <span t-out="amount_by_group['tax_group_name']">Tax 15%</span>
-                             on 
-                            <span class="text-nowrap" t-out="amount_by_group['formatted_tax_group_base_amount']">27.00</span>
+                            <span class="text-nowrap">
```

Comment:
For the next reviewer: the use of the span is needed for the class to be applied. For the span inside, it is needed for studio modification i think :smile: